_emails/us/oregon/portland_defund_ppb.md

```diff
@@ -17,34 +17,48 @@ recipients:
-  We anticipate that PPB will attempt a similar increase this year, pushing their
-  budget even higher than the proposed $245 million, thus we stand even more firmly
-  against this increase. \n\nAdditionally and critically, Wheeler's budget ELIMINATES
-  $1.8 million to fund the police body camera program. More money for policing but
-  less money for police accountability is completely unacceptable.\n\nThe police budget
-  increases regularly while other services are cut. I can see the ways in which policing
-  continues to perpetuate violence against Black, brown, houseless people, and against
-  people in mental health crisis. While Wheeler's proposal represents a slight decrease
-  from the PPB proposal, the PPB is a department whose reach into Portland communities
-  needs to be curtailed through reducing the budget, not by increasing it.\n\nMore
-  money for policing but less money for police accountability is completely unacceptable
-  at this moment in history. The city, the state, and country are all watching. And
-  so am I. I call on the Mayor, the Commissioners, and all other City Council Members
-  to use their powers to continue to reduce the budget and pledge to increase Police
-  accountability and transparency. I promise that you will lose my support and my
-  vote if you aren't willing to make a public and vocal stand to help defund the PPB
-  and increase their accountability and transparency.\n\nThank you for your time,\n[YOUR
-  NAME]\n[YOUR ADDRESS]\n[YOUR EMAIL]\n[YOUR PHONE NUMBER]"
+body: "Hello,
+
+My name is [YOUR NAME] and I am a resident of Portland. I would like to see us as a city committed to disbanding the Portland Police Bureau (PPB) immediately. We must get over this hurdle so that we can start to make real change.
+
+I am happy to see the proposed cuts in over $15 million to PPB championed by Commissioner Hardesty, and at this time I need to see more. I see the collective call for $50 million in reductions and believe this is insufficient- I want to see an immediate $100 million reduction to the PPB in the 20-21 FY budget cycle. Now is the time to take decisive action, not next year or the next, when public scrutiny will perhaps be less intense. Set a course for the full disbandment of the Portland Police Bureau NOW, or I fear it will never happen.
+
+I am skeptical of Wheeler's ability to steer us clearly as city mayor with his position at the helm of the PPB as Commissioner. The truth is, we need cuts and a commitment to disband, like the cities we see before us setting the stage for such change at this moment in time.
+
+We need to know that we are being taken seriously and that we mean for the PPB to be dismantled in support of building Black futures and community care. The PPB does not protect us equally and we can do better.
+
+Specifically, in accordance with the demands of Care Not Cops and the design to defund and reallocate, I ask for:
```

Comment:
```suggestion
Specifically, in accordance with the design to defund and reallocate, I ask for:
```

Reply:
refrain from endorsing outside organizations

Reply:
My understanding is that letters should refrain from endorsing outside orgs without permission of said org. Is this a general rule (which I see the logic of), or would it be ok to keep this given specific written permission from Care Not Cops?

Reply:
It is a general rule